typings/single-spa.d.ts

```diff
@@ -0,0 +1,99 @@
+declare module "single-spa" {
+  // ./start.js
+  export function start(): void;
+  export function isStarted(): boolean;
+
+  // ./jquery-support.js
+  export function ensureJQuerySupport(jQuery?: JQuery): void;
+
+  // ./applications/timeouts.js
+  export function setBootstrapMaxTime(
+    time: number,
+    dieOnTimeout?: boolean
+  ): void;
+  export function setMountMaxTime(time: number, dieOnTimeout?: boolean): void;
+  export function setUnmountMaxTime(time: number, dieOnTimeout?: boolean): void;
+  export function setUnloadMaxTime(time: number, dieOnTimeout?: boolean): void;
+
+  type timeoutConfig = {
+    millis: number;
+    dieOnTimeout: boolean;
+  };
+
+  type globalTimeoutConfig = {
+    bootstrap: timeoutConfig;
+    mount: timeoutConfig;
+    unmount: timeoutConfig;
+    unload: timeoutConfig;
+  };
+
+  export function reasonableTime(
+    promise: Promise<any>,
+    description: string,
+    timeoutConfig: timeoutConfig
+  ): void;
+
+  export function ensureValidAppTimeouts(
```

Comment:
This function is not exported from single-spa.

Reply:
You're right, removed